src/nrtk_explorer/library/nrtk_transforms.py

```diff
@@ -1,15 +1,13 @@
 from typing import Any, Optional, Dict, Tuple
 
+import numpy as np
+from PIL import Image as ImageModule
 from PIL.Image import Image
-
```

Comment:
pip install does not find `darkCurrentFromDensity` we might be missing a dependency/variant in pyproject.toml

Reply:
Seems that the latest `nrtk` toolkit version change where that function lives.  If you update that dependency are you still seeing a problem?

Reply:
Good catch! yes updating nrtk resolved this.